Library/Homebrew/resource.rb

```diff
@@ -98,7 +98,7 @@ def unpack(target = nil)
         yield ResourceStageContext.new(self, staging)
       elsif target
         target = Pathname.new(target) unless target.is_a? Pathname
-        target.install Dir["*"]
+        target.install Dir.glob("*", File::FNM_DOTMATCH) - %w[. ..]
```

Comment:
Can we use the following snippet instead? I think it's easier to understand, doesn't require filtering `.` and `..`, and we nowadays prefer `Pathname` over strings for path-related code.

```ruby
target.install Pathname.pwd.children
```